gearcmd/worker.go

```diff
@@ -40,11 +40,17 @@ func (conf TaskConfig) Process(job baseworker.Job) ([]byte, error) {
 	}
 	if err != nil {
 		data["error_message"] = err.Error()
-		log.Printf(kayvee.FormatLog("gearcmd", "error", "END_WITH_ERROR", data))
+		data["type"] = "gauge"
+		data["value"] = 0
+		data["success"] = false
```

Comment:
Have we confirmed with @Clever/infrastructure that this will allow alerting in librato? I'm slightly worried about the `job_id` and `job_data` being included as part of the `data` - I think that librato puts all the values together in the key name, which would make every job show be its own series with a single data point, I think.